firmware/compile.sh

```diff
@@ -1,10 +1,6 @@
 #! /bin/sh
 
-docker create -it --name env buildenv bash
+docker create -it -v $(pwd):/workspace/firmware --name env buildenv bash
```

Comment:
On my system I need to use `${pwd}` rather than `$(pwd)` - we should try to accommodate both

Reply:
fixed